src/components/tiles/text/text-tile.tsx

```diff
@@ -272,24 +267,11 @@ export default class TextToolComponent extends BaseComponent<ITileProps, IState>
       content.setSlate(value);
       this.setState({
         value,
-        selectedButtons: this.getSelectedIcons(value).sort()
+        valueRevision: this.state.valueRevision + 1
       });
```

Comment:
Good point. I'll try to address this in a follow up PR. Briefly it'd be nice to not have to use a prop on the TextTileContent since the toolbar buttons are really watching the editor content to see the selection instead of looking at something in the TextTileContent.